ibm_catalog.json

```diff
@@ -23,15 +23,15 @@
         "terraform",
         "solution"
       ],
-      "short_description": "Deploy Red Hat OpenShift cluster on Virtual Private Cloud (VPC) pre-configured with AI capabilities for accelerated machine learning workloads.",
+      "short_description": "Efficiently deploy Red Hat OpenShift cluster on Virtual Private Cloud (VPC) pre-configured with AI capabilities and GPU-enabled cluster to accelerated machine learning workloads.",
       "long_description": "This automation streamlines the deployment of Red Hat OpenShift cluster on IBM Cloud with the OpenShift AI add-on. It accelerates the setup of machine learning environments by leveraging the capabilities of the OpenShift AI add-on on Virtual Private Cloud (VPC) infrastructure with GPU-enabled clusters, reducing the complexity of AI infrastructure management and allowing data scientists and developers to focus on model development and deployment. Learn more about [Red Hat OpenShift on IBM Cloud](https://cloud.ibm.com/docs/openshift?topic=openshift-getting-started) and the [OpenShift AI add-on](https://cloud.ibm.com/docs/openshift?topic=openshift-ai-addon-about). Refer [here](https://cloud.ibm.com/docs/containers?topic=containers-supported-cluster-addon-versions#openshift-ai-416) for supported AI addon versions.",
       "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-ocp-ai/blob/main/README.md",
       "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-ocp-ai/main/images/rh_ocp_ai.svg",
       "provider_name": "IBM",
       "features": [
         {
-          "title": "AI-enhanced Red Hat OpenShift Deployment",
-          "description": "Automates the provisioning of a fully configured Red Hat OpenShift cluster on IBM Cloud with the OpenShift AI add-on. This includes optimized configurations for machine learning frameworks and simplified deployment of AI-driven applications. Refer this [documentation](https://cloud.ibm.com/docs/openshift?topic=openshift-ai-addon-about) to better understand the service and its capabilities."
+          "title": "Optional Integrations",
```

Comment:
You can mention about 
 - Openshift AI addon - Some text about it.
 - `Openshift Cluster` here that Cluster will be created where OCP AI addon will be installed.
 This is how it is done for Key Protect- 
 
<img width="992" alt="image" src="https://github.com/user-attachments/assets/b326d746-a58b-4b19-97a5-539a9fd2fba7" />